frontend/pages/g/_groupSlug/r/create.vue

```diff
@@ -57,6 +57,12 @@ export default defineComponent({
         text: i18n.tc("recipe.import-from-html-or-json"),
         value: "html",
       },
+      {
+        // just textarea dump voice actived icon
+        icon: $globals.icons.text,
+        text: i18n.tc("recipe.import-from-plaintext"),
+        value: "plaintext",
+      },
```

Comment:
Can we add a conditional to hide this if OpenAI isn't enabled? You can add a ref above this, where we already check for `enableOpenAIImages`:

```js
const enableOpenAI = computed(() => appInfo.value?.enableOpenai);
const enableOpenAIImages = computed(() => enableOpenAI.value && appInfo.value?.enableOpenaiImageServices);
```

```suggestion
        value: "plaintext",
        hide: !enableOpenAI.value,
      },
```